client/assets/stylesheets/shared/_extends-forms.scss

```diff
@@ -82,3 +90,28 @@
 		border-color: var(--color-error-dark);
 	}
 }
+
+%form-field-core-styles {
+	@include input-control;
+
+	box-sizing: border-box;
+	width: 100%;
+	height: $grid-unit-50;
+	border-color: var(--color-neutral-10);
+	font-size: $font-body-small;
+	color: var(--color-neutral-60);
+
+	@include break-small {
+		font-size: $font-body-small;
+	}
+
+	@extend %form-field-disabled;
```

Comment:
Core styles don't seem to include disabled state, so extending the existing

Reply:
Ah I see now; styles are added to the control container which we don't have here using the existing form controls. I'll see if I can also use a mixin for that.

Reply:
These styles are set in JS, so I've had to copy them in, see [cf76046](https://github.com/Automattic/wp-calypso/pull/103616/commits/cf76046f127673b3d9069f98512020b37b805cde)